pyopenetl/operations.py

```diff
@@ -351,6 +392,11 @@ def seed_table(self, read_table: str, read_chunksize: int, write_table: str) ->
             gc.collect()
 
             its += 1
+        index_creation_query = self.get_indices_from_heroku(
+            read_table=read_table, write_table=write_table, schema="public"
+        )
+        with self.dest_conn.connect() as cloud_sql_conn:
```

Comment:
Do we wanna do any error handling here in case this fails? The errors that come from SQLAlchemy here can sometimes be pretty cryptic